roles/edpm_logrotate_crond/defaults/main.yml

```diff
@@ -34,9 +34,18 @@ edpm_logrotate_crond_image: "quay.io/podified-antelope-centos9/openstack-cron:cu
 edpm_logrotate_crond_config_use_ansible: true
 edpm_logrotate_crond_config_dir: /var/lib/config-data/ansible-generated/crond
 edpm_logrotate_crond_volumes:
-  - /var/lib/kolla/config_files/logrotate_crond.json:/var/lib/kolla/config_files/config.json:ro
-  - "{{ edpm_logrotate_crond_config_dir ~ ':/var/lib/kolla/config_files/src:ro' }}"
-  - /var/log/containers:/var/log/containers:z
+  - path: /var/lib/kolla/config_files/logrotate_crond.json
```

Comment:
We need to render out these kolla files now that we're not relying on `edpm_container_manage`:

```
Mar 27 02:13:57 edpm-compute-1 edpm-compute@logrotate[36629]: Error: failed to create volume "logrotate-kolla-config": in parsing HostPath in YAML: faccessat /var/lib/kolla/config_files/logrotate_crond.json: no such file or directory
Mar 27 02:13:57 edpm-compute-1 systemd[1]: edpm-compute@logrotate.service: Main process exited, code=exited, status=125/n/a
```